Makefile.nxdk

```diff
@@ -0,0 +1,6 @@
+XBE_TITLE = ZombieBreakoutX
+GEN_XISO = $(XBE_TITLE).iso
+SRCS += $(wildcard $(CURDIR)/*.c)
+NXDK_DIR = $(CURDIR)/../..
+NXDK_SDL = y
+include $(NXDK_DIR)/Makefile
```

Comment:
Check my hode port for how to copy files to the output. You'll want to copy the necessary graphics